recipe/activate.sh

```diff
@@ -0,0 +1,38 @@
+# Path to telemac root dir
+export HOMETEL=$CONDA_PREFIX
+# Adding python scripts to PATH
+# Configuration file
+if [[ $(uname) == Linux ]]; then
+   export SYSTELCFG=$HOMETEL/configs/systel.cfg
+   # Name of the configuration to use
+   export USETELCFG=gnu.dynamic
+fi
+#OSX
+if [[ $(uname) == Darwin ]]; then
+   export SYSTELCFG=$HOMETEL/configs/systel.macos.cfg
+   # Name of the configuration to use
+   export USETELCFG=gfort-mpich
+fi
+### Python
+# To force python to flush its output
+export PATH=$HOMETEL/scripts/python3:$PATH
```

Comment:
I think PATH should be saved and restore in deactivate
Same for pythonpath and LD_lib

Reply:
could you elaborate?

Reply:
Here, you add $HOMETEL/scripts/python3 to PATH during activate
In deactivate, PATH is left untouched so $HOMETEL/scripts/python3 will stay in PATH even if you deactivate conda env.

Reply:
not sure about this.. @pmav99, what do you think when it comes to stacking multiple environments together?

Reply:
Hello both - can we not use the name "telemac-mascaret" but instead "opentelemac" please ? MASCARET will be re-integrated within TELEMAC, as one of its many modules. Great work !
Sébastien.

Reply:
Thanks Seb. 
We are directly mirroring from https://gitlab.pam-retd.fr/otm/telemac-mascaret
That's why it made sense for us to keep the same name.

Reply:
> not sure about this.. @pmav99, what do you think when it comes to stacking multiple environments together?

See https://github.com/conda-forge/qgis-feedstock/blob/main/recipe/scripts/deactivate.bat

Reply:
> not sure about this.. @pmav99, what do you think when it comes to stacking multiple environments together?

I agree that deactivate should try to restore the state of the environment. 

When you stack conda environments you do get some extra env variables. E.g. `CONDA_SHLVL`, `CONDA_PREFIX_1`, `CONDA_STACKED_2` but I would assume that conda does the correct thing by default. I.e. when you are deactivating the "inner" conda environment you should end up in the "outer" environment. But it wouldn't hurt if you do check it though.